test/FsAutoComplete.IntegrationTests/TestHelpers.fsx

```diff
@@ -275,11 +263,6 @@ let writeNormalizedOutputWith additionalFn (fn: string) (s: string) =
                                sprintf "\"%s/[^\"]*?/([^\"/]*?\.dll\")" driveLetterRegex,
                                "\"<absolute path removed>/$1")
 
-    // replace quoted paths '<path>' with <absolute path removed>
```

Comment:
we don't actually have single-quote paths in the test output anymore, so this is unnecessary